ratatool-scalacheck/src/test/scala/com/spotify/ratatool/scalacheck/AvroGeneratorTest.scala

```diff
@@ -95,18 +103,30 @@ class AvroGeneratorTest extends AnyFlatSpec with Matchers with ScalaCheckPropert
       .name("javaStringField").`type`().stringBuilder().prop(GenericData.STRING_PROP, "String").endString().noDefault()
       .name("charSequenceField").`type`().stringBuilder().prop(GenericData.STRING_PROP, "CharSequence").endString().noDefault()
       .name("utf8Field").`type`().stringBuilder().prop(GenericData.STRING_PROP, "CharSequence").endString().noDefault()
+      .name("stringableBigDecimalField").`type`().stringBuilder().prop(SpecificData.CLASS_PROP, classOf[java.math.BigDecimal].getName).endString().noDefault()
+      .name("stringableDefaultField").`type`().stringBuilder().prop(SpecificData.CLASS_PROP, classOf[StringableCustom].getName).endString().noDefault()
```

Comment:
`avsc` equivalent for
```json
{"type": "string", "java-class": "com.spotify.ratatool.scalacheck.StringableCustom"}
```
Called default because no custom `stringableGen` passed for this class